mappings/net/minecraft/nbt/CompoundWrapper.mapping

```diff
@@ -11,3 +11,4 @@ CLASS cee net/minecraft/nbt/CompoundWrapper
 		ARG 0 tag
 	METHOD c markDirty ()V
 	METHOD d getDirty ()Z
+	METHOD e getBaseTag ()Ljava/lang/String;
```

Comment:
I'm not sure if CompoundWrapper is the best name here. Not sure about getBaseTag either. Could someone look into it?